config/rbac/manager_role.yaml

```diff
@@ -0,0 +1,188 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: controller-manager
+  namespace: kcp-system
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - update
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - cert-manager.io
+    resources:
+      - certificates
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - cert-manager.io
+    resources:
+      - issuers
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - networking.istio.io
+    resources:
+      - gateways
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - networking.istio.io
+    resources:
+      - virtualservices
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - update
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - kymas
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - kymas/finalizers
+    verbs:
+      - update
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - kymas/status
+    verbs:
+      - get
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - manifests
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - manifests/finalizers
+    verbs:
+      - update
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - manifests/status
+    verbs:
+      - get
+      - patch
+      - update
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - modulereleasemetas
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - moduletemplates
+    verbs:
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - moduletemplates/finalizers
+    verbs:
+      - update
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - watchers
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
Is the watcher CR really created and deleted by lifecycle-manager? I thought this is coming from argo.